src/core_modules/capture-core/components/WidgetsChangelog/common/hooks/useChangelogData.js

```diff
@@ -32,22 +31,35 @@ export const useChangelogData = ({
         setPageSize(newPageSize);
     };
 
+    const filterParam =
+        filterValue !== 'Show all' && attributeToFilterBy
```

Comment:
The value sent from [ChangelogFilterBar](https://github.com/dhis2/capture-app/pull/3934/files#diff-63ed650bcb52cf6451cd08c69c511dcf301bc0757a0cae3f852e1b8de7bbd597R38) is `SHOW_ALL`, so this check does not work correctly. The const initialization on line [24](https://github.com/dhis2/capture-app/pull/3934/files#diff-ea0380fbff2dec66566ac55dc52502b84d62564113f532d6029fb0ea62186ab1R24) should probably be changed to `const [filterValue, setFilterValue] = useState('SHOW_ALL');`